packages/mobile/src/screens/signon/SignOn.tsx

```diff
@@ -379,8 +379,12 @@ const SignOn = ({ navigation }: SignOnProps) => {
   useEffect(() => {
     if (signedIn && accountUser) {
       setIsWorking(false)
-      setEmail('')
-      setPassword('')
+
+      // Debounce resetting state
+      setTimeout(() => {
```

Comment:
Do we need to cancel this in a cleanup task?

Reply:
i think we always want to do this, so probs not?

Reply:
well it's just a matter of ensuring we dont call 'setState' on an unmounted component, minor react thing, but good practice

Reply:
oh gotcha, will put up PR, yea